### PR TITLE
Update solution to include the correct import statements

### PR DESCRIPTION
**Pull request by sik-flow**
screen shot 2018-11-28 at 12 18 18 pm

Pull request fixes the above issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -585,9 +585,9 @@ Interestingly the most problematic variable in terms of skewness seems to be wee
 np.log(walmart["Weekly_Sales"]).describe()
 ```
 
-    /Users/lore.dirick/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:1: RuntimeWarning: divide by zero encountered in log
+    /Users/jeffreyherman/anaconda3/envs/learn-env/lib/python3.6/site-packages/ipykernel_launcher.py:1: RuntimeWarning: divide by zero encountered in log
```
Comment (PeterBell) [inline README.md]: Are we sure we want this in the notebook? Just confirming @sik-flow

File: README.md [hunk 2]
```diff
@@ -585,9 +585,9 @@ Interestingly the most problematic variable in terms of skewness seems to be wee
...
       """Entry point for launching an IPython kernel.
-    /Users/lore.dirick/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:1: RuntimeWarning: invalid value encountered in log
+    /Users/jeffreyherman/anaconda3/envs/learn-env/lib/python3.6/site-packages/ipykernel_launcher.py:1: RuntimeWarning: invalid value encountered in log
```
Comment (PeterBell) [inline README.md]: Are we sure we want this in the notebook? Just confirming @sik-flow

File: README.md [hunk 3]
```diff
@@ -632,7 +632,7 @@ walmart_log= walmart[walmart["Weekly_Sales"]>0]
 walmart_log["Weekly_Sales"]= np.log(walmart_log["Weekly_Sales"])
 ```
 
-    /Users/lore.dirick/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:1: SettingWithCopyWarning: 
+    /Users/jeffreyherman/anaconda3/envs/learn-env/lib/python3.6/site-packages/ipykernel_launcher.py:1: SettingWithCopyWarning:
```
Comment (PeterBell) [inline README.md]: Are we sure we want this in the notebook? Just confirming @sik-flow

File: README.md [hunk 4]
```diff
@@ -457,19 +457,19 @@ for idx, val in enumerate(col_names):
 
     Walmart: Weekly_Sales~Size
     ------------------------------
-    ['Size', 0.08577198301194688, 17223.235590817574, 7406.227377928731, 0.0]
+    ['Size', 0.08577198301194788, 17223.235590817574, 7406.227377928995, 0.0]
```
Comment (PeterBell) [inline README.md:460]: Is all the data supposed to be a little different?

File: README.md [hunk 5]
```diff
@@ -1566,6 +1566,13 @@ X.head()
 Let's create a for loop using `RFE` where we look at the 5, 15, 25,... up until 85 best features to be selected according to the feature ranking algorithm. Store the R-squared and the adjusted-R-squareds for all these models in a list. What do you see? No need to perform a train-test-split for now- that will be next!
 
 
+```python
+from sklearn.feature_selection import RFE
+from sklearn.linear_model import LinearRegression
+linreg = LinearRegression()
+```
+
+
```
Comment (PeterBell) [inline README.md:1561]: This looks like the heart of the change - right?!

File: README.md (hunk 4 above)
Comment (sik-flow) [reply README.md:460]: It is the same to 10 decimal points, I believe that is just the difference in how python rounds numbers

File: README.md (hunk 5 above)
Comment (sik-flow) [reply README.md:1561]: Yes and there was no import statement for RFE

**Comment by PeterBell** [comment]
LGTM - merging - thanks @sik-flow ! :+1:

**Pull request merged**